src/index.html

```diff
@@ -4,10 +4,283 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="icon" href="images/favicon.png">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header">
+      <div class="container">
+        <div class="box">
+          <div class="header__top ">
+            <div class="header__phone">+1 234 555-55-55</div>
+            <div class="header__call">
+              <img class="header__call-img"
+                src="../src/images/Vector.svg"
+                alt="call">
```

Comment:
перенось атриути правильно, якщо є три і більше атрибутів в елемента то всі потрібно переносити на нову лінію
```suggestion
              <img
                class="header__call-img"
                src="../src/images/Vector.svg"
                alt="call"
              >
```